### ISSUE TITLE
[Tooltip] Directive With Options variant is using ID of non-existing tooltip for `aria-labelledby`

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3035]
## Current behavior

Button without text label is using tooltip content as its accessible name. Accessible name is added using `aria-labelledby`.
As lazy tooltip is used in this example tooltip is not rendered in HTML code before user interacts with button. Also `aria-labelledby` is being added to button only after the tooltip is rendered in the code. 
This can result in scenerio in which screen reader user want to list all the buttons on the page before interacting with any of it. They use on VoiceOver command 'Control + Option + U' and with arrows get to list of all buttons ('Form controls') and see that element that uses lazy loaded tooltip is only announced as 'button' without any accessibility name. This scenerio is shown on screen recording.

https://github.com/user-attachments/assets/066335ce-c672-42b5-be62-b67e4a67e25e

Lack of `aria-labelledby` while tooltip is not rendered in code also causes issue "ARIA command name" reported by a11y add-on.

<img width="993" height="256" alt="Image" src="https://github.com/user-attachments/assets/454afa55-77e6-488b-9240-f8b92a3554ab" />

Additionally there is typo in story description in word "how" which now is written as "hot".

<img width="1001" height="49" alt="Image" src="https://github.com/user-attachments/assets/f0fcbf66-d18d-4417-9fc4-2769ec08e14c" />

## Sugested solution

It's recommended to replace existing lazy loading tooltip with its variant which is rendered at the same time as the button. Button should always have added `aria-labelledby` with a value of tooltip's ID. It should work as in [Panel - Actions With Tooltips](https://storybook.sanomalearning.design/?path=/story/layout-panel--actions-with-tooltips) variant.

https://github.com/user-attachments/assets/abaf3628-0768-4969-8f7b-748eda24fa98

To check if solution fix the issue turn on VoiceOver and without interacting with button use "Control + Option + U"
 and with arrows select "Form controls". There should be information about button's name and role.

Additionally check if a11y add-on is not reporting an issue  "ARIA command name".

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
